modules/local/eggnog/download.nf

```diff
@@ -37,7 +37,8 @@ process EGGNOG_DOWNLOAD {
 
     """
 
-    touch ./eggnog/eggnog.db
+    touch ./eggnog/eggnog.db.gz
```

Comment:
What's the `touch` for? Touching will produce an empty file if it's not present already, which will make the unzipping to fail (gzipped empty files are not empty). I think that if the `touch` is there to make sure there's a file one would have to do this with an `if` to circumvent this problem.